tool/dev.dart

```diff
@@ -26,7 +26,7 @@ main(List<String> args) async {
   config.copyLicense.directories = dirs;
   config.coverage.pubServe = true;
   config.format.paths = dirs;
-  config.test.platforms = ['vm', 'content-shell'];
```

Comment:
part of the CI changes... removed content shell because it's gone in Dart 2 and it avoids having to install all the fonts.